filament/backend/include/backend/DriverEnums.h

```diff
@@ -193,10 +193,12 @@ static constexpr size_t SHADER_MODEL_COUNT = 3;
  */
 enum class PrimitiveType : uint8_t {
     // don't change the enums values (made to match GL)
-    POINTS      = 0,    //!< points
-    LINES       = 1,    //!< lines
-    TRIANGLES   = 4,    //!< triangles
-    NONE        = 0xFF
+    POINTS         = 0,    //!< points
+    LINES          = 1,    //!< lines
+    LINE_STRIP     = 3,    //!< line strip
+    TRIANGLES      = 4,    //!< triangles
+    TRIANGLE_STRIP = 5, //!< triangle strip
+    NONE           = 0xFF
```

Comment:
We should also update the [Java](https://github.com/google/filament/blob/eaefaf8e59cc1255c1dd14ec5a82797f2d083b15/android/filament-android/src/main/java/com/google/android/filament/RenderableManager.java#L96-L99) and [webgl](https://github.com/google/filament/blob/eaefaf8e59cc1255c1dd14ec5a82797f2d083b15/web/filament-js/jsenums.cpp#L116-L120) bindings.

Reply:
I will search for this, totally forgot about these bindings as I'm only using filament from C++